third_party/musl/arch/aarch64/bits/alltypes.h

```diff
@@ -11,10 +11,14 @@
 #define __LONG_MAX 0x7fffffffffffffffL
 
 #ifndef __cplusplus
+#if defined(USE_COBALT_CUSTOMIZATIONS)
+typedef __WCHAR_TYPE__ wchar_t;
+#else 
 #if defined(__NEED_wchar_t) && !defined(__DEFINED_wchar_t)
```

Comment:
Couldn't we simply pre-define either `__NEED_wchar_t` to be false and provide the definition from build side ?

Reply:
I did that here: https://lbshell-internal-review.git.corp.google.com/c/cobalt_src/+/260401

however , i think this is a better solution as this fixes the problem at the source rather than relying on the platform to set flags in GN.

Another thing is this was caught because of a compile assert : if this isn't triggered on other platforms needing wchar to be 2 bytes : htat could cause the problem to go unnoticed/  if a future platform 2 years down the line relies on this code , it wouldn't be very intuitive to add those flags , fixing the source code in musl IMO fixes it for all platforms : current and future.

Reply:
Yeah, the main concern is rebasing, but as this is clearly marked change and is small I think it is OK to change musl.

Reply:
If you don't have a strong preference @y4vor  , i would strongly lean towards adding a new GN musl config which gets included as per my previous comment - just seems more maintainable in the long run IMHO

Reply:
OK, sure